### ISSUE TITLE
Positioning floating toolbar in right-hand corner

### ISSUE THREAD

**Issue by jimbobmcgee** [HenriWahl/Nagstamon#628]
Haven't been using this long, so apologies if this is an easy one and I've just missed it, but...

Is there any way to dock the floating toolbar to the right-hand side of the screen, in the Windows release?

When I position it there manually, it is dependent on the text length, which stretches left-to-right -- when there are fewer failing services there is a gutter between the edge of the toolbar and the edge of the screen.  

Is there any way to make this stretch/align from right to left; or to ensure all the segments of the toolbar are always drawn (and always a fixed size), so the toolbar will always appear "docked" to the right-hand side?

**Comment by HenriWahl** [comment]
No, you do no not miss anything - Nagstamon is missing this feature. Sounds like a useful idea. I think there was a request some years ago too.
Well, I'll keep this issue open - just in case.
